### DISCUSSION TITLE
error: local variable is never mutated

### DISCUSSION THREAD

**Discussion by ivanstepanovftw** [zigtools/zls#1968] [Ideas]
Is is possible to add "autofix" to use variable like:
```zig
var a = 0;
_ = &a; // autofix
```

Currently, what it does is:
```zig
var a = 0;
_ = a; // autofix
```

**Comment by WillLillis** [comment]
A proposal for this was rejected, as `_ = &x` is a dirty trick and shouldn't be encouraged. mlugg left a nice explanation in https://github.com/zigtools/zls/pull/1949
